src/store/validation/validationHelpers.ts

```diff
@@ -99,8 +99,8 @@ function getErrorMessageLocalizationKeyForProperty(keyPrefix: string, ajvErrorOb
   }
 
   property = property.replaceAll('/', '-')
-    // Replacing digits to eliminate resource indexes
-    .replaceAll(/\d/g, 'x')
+    // Replacing digits to eliminate resource indexes, multiple digits number (Eg: 10, 11, etc) should be replaced with 01 'x'
+    .replace((property.match(/\d+/) ?? '') as string, 'x')
```

Comment:
this fixes the validation pbm which gave us `xx` for number `10` for eg. Now, with the fix, 10 will give one single `x`

Reply:
great observation 💯